resources/charts/bitcoincore/values.yaml

```diff
@@ -135,3 +135,6 @@ baseConfig: |
 config: ""
 
 connect: []
+blocksDownload:
+  enabled: false
+  url: "https://storage.googleapis.com/warnet-signet-chains/0014d33b6e11ca95c4edccd8e986434358d79e919730/premined-signet.tar.gz"
```

Comment:
this kind of hard-code is ok in a test but i dont think it belongs in a default

Reply:
agreed, added "do not merge" to the title of the PR and added this as a todo in the description, will try to fix tomorrow.